cdc/api/v2/changefeed_test.go

```diff
@@ -321,7 +321,7 @@ func TestUpdateChangefeed(t *testing.T) {
 	cp.EXPECT().IsOwner().Return(true).AnyTimes()
 
 	// case 1 invalid id
-	invalidID := "#Invalid_"
+	invalidID := "Invalid_#"
```

Comment:
[url.Parse](https://github.com/golang/go/blob/master/src/net/url/url.go#L468) would consider `#` a fragment separator  and cut the url path into `/api/v2/changefeeds/` instead of `/api/v2/changefeeds/#Invalid` that we expected.